database/models.py

```diff
@@ -0,0 +1,46 @@
+from sqlalchemy.ext.declarative import declarative_base
+from sqlalchemy import Column, Integer, Text
+
+Base = declarative_base()
+
+
+class Report(Base):
```

Comment:
add code-based documentation for the class.
class Report(Base):
    """
    Class description
    """

Reply:
Sure, this is a remaining task in this PR